src/open_petro_elastic/config/model.py

```diff
@@ -2,16 +2,15 @@
 from typing import Optional, Union
 
 import numpy as np
-from pydantic import parse_obj_as
-from pydantic.dataclasses import dataclass
-from typing_extensions import Literal
-
 from open_petro_elastic.material import Material, hashin_shtrikman_walpole
 from open_petro_elastic.material.sandstone import (
     friable_sand,
     murphy_coordination_number,
     patchy_cement,
 )
+from pydantic import parse_obj_as
+from pydantic.dataclasses import dataclass
+from typing_extensions import Literal
```

Comment:
Why move these? According to the python overlords the previous order is the preferred.

Reply:
Fixed, just had isort configured incorrectly.